torchvision/csrc/cpu/image/readpng_cpu.cpp

```diff
@@ -0,0 +1,82 @@
+#include "readpng_cpu.h"
+
+#include <ATen/ATen.h>
+#include <png.h>
```

Comment:
same question here, shouldn't the include be only in the case we found  `libpng`?